### ISSUE TITLE
CSS prettifying Error with styling placeholder.

### ISSUE THREAD

**Issue by vajahath** [beautify-web/js-beautify#968]
Prettifying css removes a space from the code.
The code below is used to style placeholders.

**working code**
```
.card-blue ::-webkit-input-placeholder {
	color: #87D1FF;
}
``` 
**incorrect code after prettification** (Notice the space)
```
.card-blue::-webkit-input-placeholder {
	color: #87D1FF;
}
```
I tested it in google chrome. I recommend you to test this in all other browsers.

**Comment by bitwiseman** [comment]
Is this CSS or something else such as SCSS/SASS/LESS?

**Comment by bitwiseman** [comment]
This should be fixed in v1.6.4  see #985
